### DISCUSSION TITLE
Student Proof Problem

### DISCUSSION THREAD

**Discussion by EpicGenX** [community/community#119492] [GitHub Education]
### Hi everyone,

I wanted to reach out to you guys
 if there was any way to fix the problem on the documentation stats page, where it says: "Make sure that your student proof is not used previously"

This is what it shows:

![Screenshot 2024-04-14 112000](https://github.com/community/community/assets/166965251/18363bb8-040e-4ea1-bc57-00ee3e1b159c)

Does anyone know how to fix this?

**Comment by markolalovic** [comment]
Hi, check out the previous discussions on how to fix this issue:

* [Your uploaded proof has already been used previously](https://github.com/orgs/community/discussions/58442)
* [BUG：Your uploaded proof has already been used previously](https://github.com/orgs/community/discussions/52990)

**Reply by samus-aran** [reply]
Also, for completeness:

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. I've included the section that I feel would best assist.

- [Your uploaded proof has already been used previously.](https://github.com/orgs/community/discussions/111352#uploaded-proof-already-used)
